services/QuillLMS/lib/tasks/temporary/users.rake

```diff
@@ -20,4 +20,13 @@ namespace :users do
 
     puts "\nUpdated #{num_updates} user emails"
   end
-end
\ No newline at end of file
+
+  task resolve_dual_google_id_and_clever_id: :environment do
+    puts "Going to resolve #{users.count} google_id and clever_id"
```

Comment:
Where does `users` come from here?  Does this actually work?

Reply:
Good point.  That will not work.